WallsAndHoles/WallsAndHoles.pro

```diff
@@ -44,6 +44,7 @@ SOURCES += \
     newmapdialog.cpp \
     meshmaterial.cpp \
     shaderprogramonelight.cpp \
+    xmltool.cpp \
```

Comment:
Not a big deal for now, but I'd prefer a different name for this class, such as 'TileMapWritter/Loader' or something. Don't change it for now though